apps/src/templates/sectionProgress/SectionScriptProgress.js

```diff
@@ -54,6 +54,7 @@ export default class SectionScriptProgress extends Component {
       levelProgress: studentLevelProgress[studentId],
       currentLevelId: null
     };
+
```

Comment:
Looks like this file and what refers to it can be deleted since the table you have now displaces more info that this component, right?

Reply:
on second thought, it looks like Brent made some good sample data in the story that we should probably port to a new story before getting rid of: https://github.com/code-dot-org/code-dot-org/blob/staging/apps/src/templates/sectionProgress/SectionScriptProgress.story.js

totally fine in a separate PR